pymc3/aesaraf.py

```diff
@@ -146,6 +148,19 @@ def change_rv_size(
         Expand the existing size by `new_size`.
 
     """
+    if new_size is None:
+        new_size = ()
+
+    # Check the dimensionality of the `new_size` kwarg
+    new_size_ndim = new_size.ndim if isinstance(new_size, Variable) else np.ndim(new_size)
```

Comment:
Very unimportant, but do we prefer to be explicit instead of just using `np.ndim` for both?